src/open_inwoner/openzaak/statuses.py

```diff
@@ -40,12 +59,10 @@ def fetch_specific_statuses(status_urls: List[str]) -> List[Status]:
     if client is None:
         return []
 
```

Comment:
Sergei: `client.schema` voor de `with parallel` zal het schema resolven en cachen, dan moet dat niet in de threadpool zelf gebeuren. might fix your issue